### PR TITLE
removing semantic UI from seven theme

### PR DESCRIPTION
**Pull request by alioso**

### CODE REVIEW

**Comment by alioso** [comment]
Since we don't need the semantic UI files to be added to our admin theme (seven) we might as well remove them as we noticed a couple bugs resulting from that. I also removed the JS (no issue there, but why load them if not needed)

File: modules/bear_necessities/bear_necessities.module [hunk 1]
```diff
@@ -110,6 +110,26 @@ function bear_necessities_filter_info() {
 }
 
 /**
+ * Implements hook_css_alter().
+ */
+function bear_necessities_css_alter(&$css) {
+  global $theme;
+  if($theme == 'seven') {
```
Comment (jodyHamilton) [inline modules/bear_necessities/bear_necessities.module]: always use a space after if

File: modules/bear_necessities/bear_necessities.module [hunk 2]
```diff
@@ -110,6 +110,26 @@ function bear_necessities_filter_info() {
 }
 
 /**
+ * Implements hook_css_alter().
```
Comment (jodyHamilton) [inline modules/bear_necessities/bear_necessities.module:113]: Include a comment on each about the purpose of this / which module this relates to

**Comment by alioso** [comment]
All fixed and ready for review

**Pull request merged**